plugin.yml

```diff
@@ -0,0 +1,13 @@
+name: vault-secrets-buildkite-plugin
+description: A buildkite plugin to map Vault secrets to Step environments variables
+author: https://github.com/jmlrt
+requirements: []
+configuration:
+  properties:
+    secret_path:
+      type: string
+    field:
```

Comment:
added the option to get the whole content of the secret in json format so we can parse it with `jq` when no `field` is specified.

Reply:
I'd prefer a one-or-all approach if field left empty, instead of complicating the plugin with arrays. If a path contains 5 fields and we need 2, a dual invocation seems cleaner and easier to maintain.